torch/_logging/_internal.py

```diff
@@ -0,0 +1,359 @@
+import logging
+import functools
+import os
+import re
+from importlib import __import__
+from typing import Dict, Set
+from dataclasses import dataclass, field
+
+DEFAULT_LOG_LEVEL = logging.WARN
+DEFAULT_FORMATTER = logging.Formatter(
+    "[%(asctime)s] %(name)s: [%(levelname)s] %(message)s"
+)
+LOG_ENV_VAR = "TORCH_LOGS"
+
+
+@dataclass
+class LogRegistry:
```

Comment:
There aren't any docs for the fields. I think the simplest way to handle this is to just give some examples of what things should look like.

Reply:
So, uh, where are the examples

Reply:
lol I realized I did this, (along with the other comment updates) didn't commit and then lost my machine. I'll add this